internal/services/network/application_gateway_resource_test.go

```diff
@@ -5803,6 +5913,11 @@ provider "azurerm" {
 resource "azurerm_resource_group" "test" {
   name     = "acctestRG-%d"
   location = "%s"
+
+  tags = {
+    "SkipASMAzSecPack" = "true",
+    "SkipNRMSNSG"      = "true"
+  }
```

Comment:
we shouldn't be committing these?

Reply:
It should only work on our internal subscription, so I think it's better to remove these.